.gitignore

```diff
@@ -1,14 +1,20 @@
+# Python builds
+.cache
 *.pyc
 *.pyo
 /.tox
+/build
+/dist
 /lark_parser.egg-info/**
+docs/_build
+docs/examples
 tags
-.vscode
+
+# Editors and tools
 .idea
-.ropeproject
-.cache
 .mypy_cache
-/dist
-/build
-docs/_build
```

Comment:
Why are you changing the order here?

Reply:
I just alphabetised and grouped here. Not particularly partial